frontend/src/components/MapView/LeftPanel/ChartsPanel/index.tsx

```diff
@@ -336,11 +387,16 @@ const ChartsPanel = memo(
         return (
           <Box
             style={{
-              height: '240px',
+              height: '50vh',
               width: '100%',
             }}
           >
             <ChartSection
+              setChartSelectedDateRange={setChartSelectedDateRange}
+              setMaxDataTicks={setMaxDataTicks}
+              chartDateRange={chartDateRange}
+              setChartDateRange={setChartDateRange}
+              chartRange={comparePeriods ? undefined : chartRange}
```

Comment:
do we need to pass in all these?

Reply:
- `chartRange` is the output of the slider, setting what appears on the chart.
- `chartMaxDateRange` and `setChartMaxDateRange` are about knowing which dataset has the largest available data range, so we can extend the others to match it.
- `setMaxDataTicks` is for setting the sliders ticks
- `setChartSelectedDateRange` is for getting the the dates which the slider has selected, so we can show then on the label. This is necessary, since we don't have the dates in this component, the datasets hold the date-value objects

Reply:
Does this question suggest some comments would be helpful? :wink: